testing/build.gradle

```diff
@@ -55,6 +55,7 @@ dependencies {
       'com.google.dagger:dagger:2.24',
       'org.jetbrains.kotlinx:kotlinx-coroutines-test:1.2.2',
       'org.robolectric:robolectric:4.3',
+      'org.jetbrains.kotlin:kotlin-reflect:$kotlin_version',
```

Comment:
Also, could you check if there is a requirement of reflection in the app module anymore or can we remove the dependency from that module?

Reply:
It wasn't required in the app module anymore. Removed it.